packages/p2p-v2/src/components/PopoverDropdown/PopoverDropdown.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useRef, useState } from 'react';
+import { Button, Text } from '@deriv-com/ui';
+import { useOnClickOutside } from 'usehooks-ts';
+import DropdownIcon from '../../public/ic-more.svg';
```

Comment:
just a note here, if my quill-icons PR gets merged, see if you can replace this icon with one from the library